actions/kpops-runner/action.yaml

```
@@ -34,10 +34,20 @@ inputs:
   token:
     description: "secrets.GITHUB_TOKEN, needed for setup-helm action if helm-version is set to latest"
     required: false
+  python-version:
+    description: "Python version"
+    required: false
+    default: "3.10.6"
```

```suggestion
    default: "3.11"
```
3.11 for better tracebacks
patch level doesn't matter for the default imo